app/js/enhancedDate.js

```diff
@@ -1,5 +1,133 @@
-/* 
- * add your module definition for enhancedDate here
- * assign the module to a global variable
- * /
+var enhancedDate = (function (date) {
+// still maintains access to all globals
+  
+  var setDate = {};
```

Comment:
You're declaring a variable setDate, and assigning it an empty object, but then immediately following you're assigning a function to the same variable. Rather than setting setDate to an object on line 4, use a different variable name. That could be the object that you put your methods on and return from your module (see comment below regarding the need to return an object from your module.)